packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -467,7 +467,7 @@ export class ConnectionManager implements IConnectionManager {
                     this.logger.sendTelemetryEvent({ eventName: "ReceivedClosedConnection" });
                     connection = undefined;
                 }
-            } catch (origError) {
+            } catch (origError: any) {
```

Comment:
Did you update all catch variables to be `: any` or just those that hit errors now that the default is `unknown`?

I don't want to encourage the `: any` behavior, so I will open an issue to clean these up after you merge this.

Reply:
I only used any where it was an error to use unknown, and I confirmed all of them were cases where it at least kind of made sense to use any (mostly tests or providing special handing for specific fields if present).

A follow-up is a great idea: I suspect several of the usages would be better handled with unknown + an instance of test, or something like that.